Main.qml

```diff
@@ -262,12 +262,15 @@ ApplicationWindow {
 
         Button {
             id: actionButton
+            // this should be set declaratively
```

Comment:
It isn't?

Reply:
Not quite... a few places set it via `PropertyChanges`